README.rst

```diff
@@ -6,13 +6,166 @@ App for tracking sprint commitments, vacations and spillovers.
+
+Cell's dashboard
+^^^^^^^^^^^^^^^^
+
+Estimations
+~~~~~~~~~~~
+The basic idea for calculating estimations is the following:
+
+1. `SPRINT_HOURS_RESERVED_FOR_MEETINGS` hours are reserved for the meetings for each sprint.
+2. `SPRINT_HOURS_RESERVED_FOR_EPIC_MANAGEMENT` hours are reserved for epic management for each sprint.
+3. 1 hour is planned for reviewing each task with <= 3 story points. For bigger tasks, 2 hours are reserved.
+4. Each of these defaults can be overridden for each ticket by putting the following in the ticket’s description:
+    a) [~{JIRA_BOT_USERNAME}]: plan `time` per sprint for epic management
+    b) [~{JIRA_BOT_USERNAME}]: plan `time` per sprint for this task
+    c) [~{JIRA_BOT_USERNAME}]: plan `time` for reviewing this task
+
+**Note**: The `time` should match the following regexp:
+
+.. code::
+
+    (?:(?P<hours>\d+)\s?h.*?)?\s?(?:(?P<minutes>\d+)\s?m.*?)?
+
+, so anything like 1h 30m, 1h or 30m will work. You can use the following directives for overriding the default values:
+
+Dashboard
+~~~~~~~~~
+For each member of the cell this view displays:
+
+1. Assignee, reviewer, upstream and epic-related hours for the present and upcoming sprint.
+2. Tickets that are missing an estimation.
+3. Scheduled vacations for the upcoming sprint, which are retrieved from the Google Calendar events that match `GOOGLE_CALENDAR_VACATION_REGEX`. Partial vacations (based on the time of the meeting for each person’s day) are retrieved from the `GOOGLE_AVAILABILITY_RANGE` of the `GOOGLE_CONTACT_SPREADSHEET` using the `GOOGLE_AVAILABILITY_REGEX`. The minus (-) suffix there indicates that the member’s timezone is UTC-X. This is taken into account if user’s workday spans over two days in UTC (i.e. start hour is after end hour).
+4. Committed time, which is the sum of all estimation hours.
+5. Goal for the next sprint, which is calculated by summing up user's commitments for each day of the sprint and subtracting `SPRINT_HOURS_RESERVED_FOR_MEETINGS` and `Vacation` hours from it.
+6. Remaining time, which is the result of `Goal` - `Committed`.
+
+User's dashboard
+~~~~~~~~~~~~~~~~
+This view shows all assigned (as `Assignee` or `Reviewer 1`) tickets of the user with:
+
+1. Task's key (you can hover over it to see the ticket's name)
+2. User's role
+3. Current status of the ticket
+4. Remaining time for the current user
+5. Sprint indicator (active or future one)
+6. Epic/Story indicator
+
+Caching
+~~~~~~~
+After you refresh the board for the second time, you’ll immediately see cached data **and a spinner showing that it’s being reloaded**. This makes using the dashboard much smoother.
+
+
+Completing the sprints
+~~~~~~~~~~~~~~~~~~~~~~
+In case when users needs to schedule tickets for sprints that haven’t been created yet, they can press the `Create Next Sprint` to create a new one for the currently viewed cell.
```

Comment:
@Agrendalath Typos:

* "In case when users needs" should be "In case a user needs".
* "press the \`Create Next Sprint\`" should be "press \`Create Next Sprint\`".